Makefile

```diff
@@ -350,13 +352,39 @@ echo-release-charts:
 .PHONY: buildah-docker
 buildah-docker:
 	./hack/bin/buildah-compile.sh
-	BUILDAH_PUSH=${BUILDAH_PUSH} ./hack/bin/buildah-make-images.sh
+	BUILDAH_PUSH=${BUILDAH_PUSH} KIND_CLUSTER_NAME=${KIND_CLUSTER_NAME} BUILDAH_KIND_LOAD=${BUILDAH_KIND_LOAD}  ./hack/bin/buildah-make-images.sh
 
 .PHONY: buildah-docker-%
 buildah-docker-%:
 	./hack/bin/buildah-compile.sh $(*)
-	BUILDAH_PUSH=${BUILDAH_PUSH} EXECUTABLES=$(*) ./hack/bin/buildah-make-images.sh
+	BUILDAH_PUSH=${BUILDAH_PUSH} EXECUTABLES=$(*) KIND_CLUSTER_NAME=${KIND_CLUSTER_NAME} BUILDAH_KIND_LOAD=${BUILDAH_KIND_LOAD} ./hack/bin/buildah-make-images.sh
 
 .PHONY: buildah-clean
 buildah-clean:
 	./hack/bin/buildah-clean.sh
+
+.PHONY: kind-cluster
+kind-cluster:
+	kind create cluster --name $(KIND_CLUSTER_NAME)
```

Comment:
Would it make sense to use a kind config file like in https://github.com/wireapp/wire-server/pull/1087/files#diff-2b5ffee781d627b4719c83f613ca08512b9cae4919718dc9f535f14ded176b48R1-R11 here?

Reply:
Also, would it make sense to lock a specific version of kubernetes here? With `--image kindest/node:vXYZ`?

Reply:
Can we do those things incrementally, as we need them? I would love to keep this as config free as possible.

Reply:
Sure, that works also.